### DISCUSSION TITLE
关于urdf文件导入webots

### DISCUSSION THREAD

**Discussion by johnning0530** [cyberbotics/webots#5358] [Q&A]
您好，我尝试将挖掘机3D模型转为urdf导入webots实现挖掘机的基本挖掘动作。但似乎挖掘机的液压元件无法处理，违背了urdf文件的串联关系。您有什么好的建议？
拆除液压元件版本的urdf导入webots后，joint或link结点也全部消失了，无法实现挖掘机仿真。
想请教您，我的项目是否无法在webots中完成呢，我或许需要通过unity实现？

**Comment by huiduan-25** [comment]
也许用英语问可能会回复的人多一点
